docker/ci/script/run.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash
 
-/usr/local/bin/docker-compose -p $1 run -v $1-gradle:/home/jenkins/.gradle -v $1-m2:/home/jenkins/.m2 --rm --user jenkins --entrypoint "$2" my-studies-builder
+USER_NAME="$(id -un)"
+USER_ID="$(id -u)"
+GROUP_NAME="$(id -gn)"
+GROUP_ID="$(id -g)"
+
+stop_and_remove_containers() {
+  /usr/local/bin/docker-compose -p $1 stop
+  /usr/local/bin/docker-compose -p $1 rm -f -v
```

Comment:
Voisi asettaa `/usr/local/bin`in PATHiin niin sitä ei tarvitsisi toistaa joka komennossa.

Lisäksi tässä pitäisi riittää sanoa: `docker-compose rm -fsv` (`-s` pysäyttää kontit)